metadata-db-app/int-test/cmr/metadata_db/int_test/provider_crud_test.clj

```diff
@@ -1,163 +1,161 @@
 (ns cmr.metadata-db.int-test.provider-crud-test
-  "Contains integration tests for deleting providers. Tests delete with various configurations including
-  checking for proper error handling."
+  "Contains integration tests for CRUD of providers. Tests CRUD with various
+   configurations including checking for proper error handling."
   (:require [clojure.test :refer :all]
-            [cmr.metadata-db.int-test.utility :as util]
-            [cmr.common.util :as u]
+            [cmr.metadata-db.int-test.utility :as i-util]
```

Comment:
I think `test-util` and `util` instead of single letter prefixes would be easier to read.